### ISSUE TITLE
input ticket - set app version on task creation

### ISSUE THREAD

**Issue by jackDiGi** [sbg/sevenbridges-python#16]
might be a nice feature to be able to pick a prior version of the app, was straightforward with the HTTP calls. **low priority**

**Comment by damirkrstanovic** [comment]
You can basically do it, but you need to know the structure of the apps.

So for example I can do something like this:
```
app = api.apps.query(limit=2, project='damirk/batch-playground')[1]
```
Then if app is `damirk/batch-playground/whole-exome-sequencing-gatk-2-3-9-lite/1`, I can do something like this:
```
old_version = api.apps.get('damirk/batch-playground/whole-exome-sequencing-gatk-2-3-9-lite/0')
```

Then I can use `old_version` to run tasks.

Versioning is strictly incremental, so if I see version 5, I can use 0-5 revisions.

Does this help @jackDiGi  ?

**Comment by SenadI** [comment]
@jackDiGi, like @damirkrstanovic described this can be actually done. You have all the information in the app.raw for all revisions. The knowledge of our revision system is needed. I can not add a revision choice to the create_task because it will break the current api. I will close this issue because there is a simple workaround like @damirkrstanovic suggested.

**Comment by SenadI** [comment]
@jackDiGi I've reopened this because with the changes I will introduce by solving [this](https://github.com/sbg/sevenbridges-python/pull/19) we can achieve this without breaking compatibility. FYI @jdagilliland - I previously closed this but with solving the app revision issue I will also solve this one. Expect a new release tomorrow :)

**Comment by SenadI** [comment]
@jackDiGi Fixed in v0.4.3, now you will be able to provide a revision during task creation.
